doc/reference/guidelines.rst

```diff
@@ -163,7 +163,8 @@ To declare a record in XML, the **record** notation (using *<record>*) is recomm
 - Try to group the record by model. In case of dependencies between
   action/menu/views, this convention may not be applicable.
 - Use naming convention defined at the next point
-- The tag *<data>* is only used to set not-updatable data with ``noupdate=1``
```

Comment:
I am not sure about removing references to the `data` tag. It is still valid and, as we discussed in a previous issue, using data is still necessary for two noupdate clause in the same file